src/jzon.lisp

```diff
@@ -1866,5 +1866,18 @@ see `write-object'"
          (with-output-to-string (stream)
            (stringify-to stream)))
         (t
-          (stringify-to stream)
-          nil)))))
+         (stringify-to stream)
+         nil)))))
+
+;; NOTE: Borrowed from UIOP.
```

Comment:
Did you check uiop's licence to see if it was ok to grab code from it?

Reply:
[Seems to be MIT](https://gitlab.common-lisp.net/asdf/asdf/-/blob/master/uiop/uiop.asd?ref_type=heads#L29), which I interpret to mean "do whatever you want" (see https://opensource.org/license/mit).